WalletWasabi.Fluent.Desktop/WalletWasabi.Fluent.Desktop.csproj

```diff
@@ -51,6 +51,7 @@
 		<AvaloniaResource Include="Assets\**" />
 	</ItemGroup>
 	<ItemGroup>
+		<PackageReference Include="System.Drawing.Common" Version="6.0.0" />
```

Comment:
It seems to me that `System.Drawing.Common` is already included without the PackageReference but an older version. See the packages.lock.json. Maybe that version is OK for as as well.

Reply:
It's actually not even needed here under the `WalletWasabi.Fluent.Desktop` project, only under `WalletWasabi.Fluent`. Im removing it. Version is good 'tho.